examples/contracts/aggregator_consumer/Scarb.toml

```diff
@@ -8,10 +8,13 @@ name = "aggregator_consumer"
 version = "0.1.0"
 cairo-version = "2.6.3"
 
+# [scripts]
+# test = "snforge test"
+
```

Comment:
We probably don't need this either right?

Reply:
yeah, we don't. i might add it though to make use of `scarb test`